pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -887,7 +887,7 @@ Block.Tabular:
         content:
           type: array
           items:
-            $ref: '#/Block.Tabular.ContentRow'
+            $ref: '#/Question'
```

Comment:
We are replacing 2-dimensional array with just a flat one here.

A question: 
perhaps, we need to put `Block.Question` here ? And add a `columnSpan` property to it (instead of the current place in `Question`)

Reply:
I think it should be Block.Question. Looking at similar things to tabular container in our api, like conditional block, they refer to blocks. example below:
https://github.com/broadinstitute/ddp-study-server/blob/674b2216522fd3d83bd1f074d0b74eaccb3ad38e/pepper-apis/docs/specification/src/components/schemas.yml#L773-L800

And if we can have the columnSpan on the Block.Question instead of the Question. Or, if we want to be ambitious, even on Block? Nah! Block.Question would be good for a long while.

Reply:
Fixed in 21eaf9b59ee2589a40edde101d0d7abea3df6517

Reply:
Great! Let's not forget to remove columnCount from Question now that Block.Question has that property:
https://github.com/broadinstitute/ddp-study-server/blob/674b2216522fd3d83bd1f074d0b74eaccb3ad38e/pepper-apis/docs/specification/src/components/schemas.yml#L1626-L1629

Reply:
You might mean `columnSpan` ? 
If yes, it is removed in 21eaf9b59ee2589a40edde101d0d7abea3df6517